_posts/2021-01-14-voicemeeter-banana.md

```diff
@@ -0,0 +1,110 @@
+
+Location of Virtual Audio Cable in Voicemeeter after installation:
+'C:\Program Files (x86)\VB\Voicemeeter'
+
+
+## Setting up Voicemeeter Banana
+
+Please open Voicemeeter Banana.
+
+### Saving Settings, Loading Settings, Resetting settings
+Something that typically gets skipped in other Voicemeeter coverage that I've seen is the ability to save settings. At any point you can save your current settings of Voicemeeter by going to Menu→Save Settings. This saves a settings file on your computer you can then Menu->Load to load any saved settings from just a file even on other devices should you want to transfer your settings to another device. If at any point you feel lost and want to restart the setup, you can go to Menu→ Reset Settings (Re-Initialization) and this will make Voicemeeter appear as if you had just installed it, blank with nothing set up.
+
+![Image for showing where the menu and save/load/reset settings are](\assets\images\voicemeeter\vm_save_load_reset_settings.webp)
+
+
+### Audio outputs
+The Hardware Outputs section in the top right should have A1, A2, A3 with a triangle beside them. Each one of these have a selectable menu for our output devices. Each input device has a toggle-able setting for us to push audio to each individual output device, which will get better explanation when we get to inputs. For now, click on A1 and select your headphones. Click on A2 and select the virtual audio cable.
+
+Your A1, A2, and A3 section should look a little something like this:
+
+![Voicemeeter Hardware Outputs](\assets\images\voicemeeter\vm_hardware_outputs.webp)
+
+Below the hardware output selection you will see a bunch of vertical bars, for now we will leave these alone until we have audio pushing through them. For that, we will start by pushing computer audio to them through a digital inputs.
+
+
+### Digital Inputs and Windows Audio
+For digital inputs we will be looking at the middle columns of Voicemeeter Banana which should look a little something like this:
+
+![Voicemeeter Digital Inputs](\assets\images\voicemeeter\vm_digital_inputs.webp)
+
+To set our Windows Default Audio to push through these digital inputs we need to open Windows Sound Settings by right clicking on the speaker icon that adjusts our audio in the task bar. After right clicking select 'Open Sound Settings', (alternatively you can open the windows start menu and type 'sound settings' and it should show up for you to open). You should see a window that with these:
```

Comment:
One thing I've seen before is to write down the default audio devices before you install VM, then after you install VM, set VM to be the default audio devices for both input and output. What are your thoughts on that?